components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2117,7 +2117,11 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
         }
     }
 
-    protected defaultSpendingLimit = 100;
+    protected defaultUsageLimits = {
+        individualFree: 500,
```

Comment:
The default spending limit for tams and users would be provided through configuration on the usage component.

Reply:
Aha. Maybe it makes sense then to not send along some value from `server`, but let the `usage` component create CostCenters with the appropriate Usage Limit as configured?

Reply:
@jankeromnes I believe that's a good approach, that allows us to centralize that logic in a single place. Happy to help with the changes if needed!